src/scripts/statusbar.ts

```diff
@@ -55,7 +65,8 @@ namespace Scripts {
         }
 
         static updateStatusBarGumpForObject(o: GameObject, s: any, gump: CustomGumpObject, forceUpdate = false) {
-            const name = o.Name();
+            let name = o.Name();
```

Comment:
zas `o` 🤦 

:D

Reply:
zivot jsou pismenka, o name tam bylo orignalne, ja sem ho pouze vyuzil... predim nikomu nevadil

Reply:
Tady to z nazvu metody a typu parametru jeste dava smysl :-) je mi to fuk ale